docs/index.txt

```diff
@@ -27,6 +27,8 @@ following pages should help you get started:
 
 - :doc:`/tutorial/install-php-library`
 
+- :doc:`/tutorial/connecting`
```

Comment:
Let's also add `/tutorial/connecting` to the `toctree` in `tutorial.txt` to ensure this page is surfaced there as well.

```diff
Tutorials
=========

.. default-domain:: mongodb

.. toctree::

+  /tutorial/connecting
   /tutorial/crud
   /tutorial/collation
```